setup.cfg

```diff
@@ -1,3 +1,7 @@
+[options]
+install_requires =
+    setuptools;python_version>'3.11'
```

Comment:
I don't like adding a dependency on `setuptools`, but most likely you want `>=3.12` here.